grid/app/__init__.py

```diff
@@ -53,7 +53,7 @@ def set_database_config(app, test_config=None, verbose=False):
     db.init_app(app)
 
 
-def create_app(debug=False, n_replica=None, test_config=None):
+def create_app(node_id: str, debug=False, n_replica=None, test_config=None):
```

Comment:
```suggestion
def create_app(node_id: str, debug=False, n_replica=None, test_config=None) -> Flask:
```
please add return type -> `Flask`